### ISSUE TITLE
Fatal error: 'try!' expression unexpectedly raised an error: Error Domain=io.realm Code=1 "fcntl() inside prealloc() failed: Invalid argument" UserInfo={NSLocalizedDescription=fcntl() inside prealloc() failed: Invalid argument, Error Code=1}: file /BuildRoot/Library/Caches/com.apple.xbs/Sources/swiftlang/swiftlang-900.0.74.1/src/swift/stdlib/public/core/ErrorType.swift, line 181

### ISSUE THREAD

**Issue by bb-git** [realm/realm-cocoa#5701]
Since upgrading to version 3.30 some of my customers have this error:

Fatal error: 'try!' expression unexpectedly raised an error: Error Domain=io.realm Code=1 "fcntl() inside prealloc() failed: Invalid argument" UserInfo={NSLocalizedDescription=fcntl() inside prealloc() failed: Invalid argument, Error Code=1}: file /BuildRoot/Library/Caches/com.apple.xbs/Sources/swiftlang/swiftlang-900.0.74.1/src/swift/stdlib/public/core/ErrorType.swift, line 181

Realm framework version: 
3.30

Xcode version: 
9.2

iOS/OSX version: 
11.2.5

Dependency manager + version: 
cocoapods 1.4

**Comment by tgoyne** [comment]
3.3.0 does include the linked PR. The changes there appear to have been insufficient.

**Comment by hyouuu** [comment]
@tgoyne thanks for the 3.3.1! However it's not showing up from CocoaPods (yet) - any steps needed or would it solve itself after a few hours?

**Comment by tgoyne** [comment]
The website deploy is currently failing, and that has to complete before we can push the new version to cocoapods.

**Comment by tgoyne** [comment]
3.3.1 is now fully released.

**Comment by hyouuu** [comment]
@tgoyne fyi on X I still get crash when adding multiple things at once, although it's possible there's a separate issue

**Comment by momodevelopment** [comment]
the same issue with me.
Realm framework version: latest 
Realm (3.2.0-beta.3), RealmSwift (3.2.0-beta.3)
Xcode version: 9.2,
iOS/OSX version: 11.2.5
Swift version 3.2

@tgoyne do you mean version 3.3.1 fix the problem? any way to get it

**Comment by k-neo** [comment]
+1

Realm 3.9.0
iOS version: 12.0.1, 11.4.1
